### DISCUSSION TITLE
Lutfi Laili Salim | Lutt

### DISCUSSION THREAD

**Discussion by lutfilaili** [onlyphantom/llm-python#30] [Practicum: Generative AI]
https://colab.research.google.com/drive/1Z7C1vBwh-uaNRJuxlVwPyjnrUIWOeW5f?usp=sharing

**Comment by onlyphantom** [comment]
Very well done Lutfi! I liked that you really worked on the prompt template, and also the fact that you've done both Challenges 1 and 2 even though only one is required. 

Excellent work sir! Marked as passed and I'll be issuing the certificate soon!

**Comment by lutfilaili** [comment]
Hi @onlyphantom (Sam),

Thank you for your assessment.
I believe there are numerous aspect I can improve from my code.
This is my very first time with Generative AI, while it was challenging, but I could feel enjoy during process.

I wish I could apply this knowledge to my company, I work in a manufacturing domain. I envision, in the future, many manufacturers in Indonesia utilize AI to streamline their works.

Thanks, I greatly appreciate your effort to introduce Gen AI to me.
Hopefully, I can follow another Gen AI workshop with you!
